tools/benchmark/memory/src/main.rs

```diff
@@ -110,12 +110,9 @@ fn main() {
         fs::read_dir("target/debug/examples")
             .unwrap()
             .flat_map(|entry| {
-                entry
-                    .ok()?
-                    .file_name()
-                    .into_string()
-                    .ok()
-                    .filter(|s| !s.contains(['-', '.']))
+                entry.ok()?.file_name().into_string().ok().filter(|s| {
+                    !dgb!(s).ends_with(".d") && s.as_bytes()[s.len().saturating_sub(17)] != b'-'
```

Comment:
is this using `dbg`? should this be removed?

Reply:
I want to see what the windows target directory actually looks like as I don't have one

Reply:
oh typo

Reply:
I guess I can also run on the PR